Source/WebCore/Modules/modern-media-controls/images/iOS/ExitFullscreen.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16px" height="16px" viewBox="0 0 16 16" version="1.1" xmlns="http://www.w3.org/2000/svg">
```

Comment:
```suggestion
<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```

Reply:
Agreed to removing the `version` attribute. I'm not sure what the benefit of removing the XML declaration is - this is valid XML after all. 

As for removing the `px` unit specifiers from the `width` and `height`… I'm not sure it matters. It would mean the value is in the current coordinate system, but that would be the same as `px`. I think it might be better to simply remove the attributes completely, but they are actually used from the media controls code to evaluate the size of the button, iirc.

Reply:
I wonder if I can push to this branch on Antoine's repository.